gen_types.go

```diff
@@ -3562,6 +3562,20 @@ type Message struct {
 	ReplyMarkup *InlineKeyboardMarkup `json:"reply_markup,omitempty"`
 }
 
+// GetLink is a helper method to easily get the message link excluding group and private chat types.
+func (m *Message) GetLink() string {
+	if m.Chat.Type != "private" && m.Chat.Type != "group" {
```

Comment:
Consider reducing indentation levels by following the principle of "fail fast".
In this case, if the type is private, or group, return the empty string.

This reduces indentation, making your code easier to read.